pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+# <pull_request_title>
+
+[Jira Ticket](<link_to_jira_ticket>)
+- [ ] Ready for review
+
+## Checklist
+- [ ] I have completed a self review
+- [ ] I have updated the [spreadsheet](https://docs.google.com/spreadsheets/d/1277nPOKtBFOk_kCo870_zA460-qlw_kGnYlLPwtnadE/edit#gid=0)
+- [ ] I have updated the jira ticket
+- [ ] I have added the relevant labels to this PR
+- [ ] I have updated documentation (if applicable)
+- [ ] I have run the unit tests suite and they pass
+- [ ] I have updated the [change log](https://docs.google.com/document/d/1t3s6lQuUcug1Fj8E_i2gBJNvXN0-funwBArG2hsia6k/edit)
+
+
+## Description
+
+## Notes
+
+## Dependencies for merge
+- [ ] [Pull request link]()
```

Comment:
If you revert to format I sent you, you can link with just #<pr num>

Reply:
Ah I did not realise that's what it was for, I will change it